src/app/index.ts

```diff
@@ -276,7 +276,7 @@ module.exports = yo.extend({
   writing: {
     copyFiles: function () {
       try {
-        let language = this.project.ts ? 'ts' : 'js';
+        let language = this.project.ts || this.project.framework === excelCustomFunctions ? 'ts' : 'js';

```

Comment:
I thought the custom functions were actually js-based today, I didn't see any Ts (unless you've recently added this?...)

Reply:
Confirmed that Custom Functions work in Excel Online after adding the following to webpack.config.js:

    devServer: {
        port: 3000,
        hot: true,
        inline: true,
        headers: {
            "Access-Control-Allow-Origin": "*"
        }
    }